pyproject.toml

```diff
@@ -83,6 +83,7 @@ select = [
   "S",      # flake8-bandit
   "SIM",    # flake8-simplify
   "SLF",    # flake8-self
+  "T10",    # flake8-debugger
```

Comment:
Why enable it if it's to explicitly disable its check with `noqa` ?

Reply:
It could be helpful to prevent newly checked in code if it accidentally contains a set breakpoint.